.github/workflows/security-hardened.yml

```
@@ -108,24 +108,24 @@ jobs:
       - name: Validate CORS configurations
         run: |
           echo "🌐 Validating CORS configurations..."
-
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)